### ISSUE TITLE
Sentry issues

### ISSUE THREAD

**Issue by joacub** [vercel/next.js#52620]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
latest canary version
```

### Link to the code that reproduces this issue or a replay of the bug

no response

### To Reproduce

no response

### Describe the Bug

 - error TypeError: Cannot destructure property 'RawSource' of 'require(...)' as it is undefined.
@wsy/web:dev:partial:docker:     at @sentry/nextjs (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/app/[lang]/admin/page.js:33:18)
@wsy/web:dev:partial:docker:     at __webpack_require__ (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/webpack-runtime.js:33:43)
@wsy/web:dev:partial:docker:     at eval (./src/app/layout.tsx:21:72)
@wsy/web:dev:partial:docker:     at (sc_server)/./src/app/layout.tsx (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/app/[lang]/admin/page.js:10142:1)
@wsy/web:dev:partial:docker:     at Function.__webpack_require__ (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/webpack-runtime.js:33:43)
@wsy/web:dev:partial:docker: - error TypeError: Cannot destructure property 'RawSource' of 'require(...)' as it is undefined.
@wsy/web:dev:partial:docker:     at @sentry/nextjs (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/app/[lang]/admin/page.js:33:18)
@wsy/web:dev:partial:docker:     at __webpack_require__ (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/webpack-runtime.js:33:43)
@wsy/web:dev:partial:docker:     at eval (./src/app/layout.tsx:21:72)
@wsy/web:dev:partial:docker:     at (sc_server)/./src/app/layout.tsx (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/app/[lang]/admin/page.js:10142:1)
@wsy/web:dev:partial:docker:     at Function.__webpack_require__ (/Users/johan/dev-projects/weshipyou-monorepo/apps/web/.next/server/webpack-runtime.js:33:43)
@wsy/web:dev:partial:docker: digest: "3742216727"

### Expected Behavior

working

### Which browser are you using? (if relevant)

no response

### How are you deploying your application? (if relevant)

no response

**Comment by mmintel** [comment]
@joacub had the same issue after update but was related to sentry  plugin

**Comment by navFooh** [comment]
Having the same issue here as well.

This configuration works fine:
- `@sentry/nextjs` at `7.58.1`
- `next` at `13.4.9`

Updating `next` to `13.4.10` causes `Cannot destructure property 'RawSource' of 'require(...)' as it is undefined`.

**Comment by grabbou** [comment]
I can't reproduce on latest Sentry - just upgraded to `7.58.1` and error went away.

**Comment by godfrednanaowusu** [comment]
> I can't reproduce on latest Sentry - just upgraded to `7.58.1` and error went away.

I can confidently say, this solves it for sure

**Comment by nimaa77** [comment]
I have these versions installed

`@sentry/nextjs: 7.58.1,
next: 13.4.10`

and get the same 'RawSource' of 'require(...)' as it is undefined. error
maybe it's related to the i18n config?

I'm using `next-intl: 3.0.0-beta.8`

**Comment by DrChrispoper** [comment]
I can confirm 
This configuration works fine:
- @sentry/nextjs at 7.58.1
- next at 13.4.9
and 
- next at 13.4.10 doesn't

So I updated from `"next": "^13.4.9"` to `"next": "13.4.9"` to deploy that specific version

**Comment by navFooh** [comment]
> try added `transpilePackages: [ '@sentry/nextjs' ]` in next.config.js

Doesn't work for me

**Comment by jknight12882** [comment]
Also experiencing this issue. Started with `next@13.3.10`

**Comment by joacub** [comment]
none of the solutions works, updating sentry does not work, there is something in the transpilers that is not working properly after 13.4.10, seems like is not locating the package properly. also there is other issues similar to this one, that can not locate the correct file

this maybe related: https://github.com/vercel/next.js/issues/52621

**Comment by HamAndRock** [comment]
Having same issues after updating from next@13.4.9 to next@13.4.10, even tried updating sentry like people are suggeting but that also didn't help.

**Comment by nclayt0n** [comment]
We were having the same issues. we found that pinning to version 13.4.9 resolved our sentry issues.

**Comment by godfrednanaowusu** [comment]
for everyone having this issue, i had to revert to versions that worked, in my case

next = 13.4.7
@sentry/nextjs =7.57.0

these versions work

**Comment by creativoma** [comment]
I also had the same problem. I had to revert the changes to a previous commit and not update Next to next@13.4.10.

**Comment by KrustyC** [comment]
same here, problem started as soon as I update to next@13.4.10 and stopped as soon as I downgrade to 13.4.9

**Comment by grabbou** [comment]
FYI, there's beta release of Sentry SDK as mentioned in https://github.com/getsentry/sentry-javascript/issues/8541, please try it out and report if it fixes the problem.

**Comment by jinsley8** [comment]
It looks like the issue occurs since `13.4.10-canary.5`

_https://github.com/getsentry/sentry-javascript/issues/8541#issuecomment-1638370521_

**Comment by GRA0007** [comment]
This issue has been fixed in Sentry release v7.59.0
https://github.com/getsentry/sentry-javascript/releases/tag/7.59.0

**Comment by emmgfx** [comment]
Anyone has been able to install the new releases? 

```
npm i @sentry/nextjs@7.59.0
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @sentry/nextjs@7.59.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

```
npm i @sentry/nextjs@7.59.1
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @sentry/nextjs@7.59.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

```
And `@latest` installs` "@sentry/nextjs": "^7.58.1", `

**Comment by DusanStojanovic-gowago** [comment]
I can confirm that it works for me:

```
"next": "^13.4.10",
"@sentry/nextjs": "^7.59.2",
```

**Comment by jonahallibone** [comment]
Is this a Next issue or a Sentry issue though? Afaik the Sentry team unsure why the 13.4.10 broke their Webpack import

**Comment by DusanStojanovic-gowago** [comment]
I believe the Next team made a change that caused the issue for Sentry. But the Sentry team found a workaround. That explains why upgrading the `@sentry/nextjs` fixed it, not vice-versa.

**Comment by Barokai** [comment]
maybe related: 13.4.10 had a change to how/when webpack is loaded: https://github.com/vercel/next.js/pull/52587

**Comment by jonahallibone** [comment]
> maybe related: 13.4.10 had a change to how/when webpack is loaded: #52587

Ah yes this is what I was curious about

**Comment by jandolezal71** [comment]
Looks lite it is fixed in `@sentry/nextjs 7.59.3` https://github.com/getsentry/sentry-javascript/pull/8584
